src/main/java/org/folio/utils/CalculationUtils.java

```diff
@@ -107,7 +111,8 @@ private static Map<String, BigDecimal> calculate(Double dInitialAllocation,
     BigDecimal totalFunding = allocated.add(netTransfers);
     BigDecimal cashBalance = totalFunding.subtract(expended);
     BigDecimal available = totalFunding.subtract(unavailable);
-    BigDecimal overExpended = expended.add(awaitingPayment).subtract(totalFunding.max(BigDecimal.ZERO)).max(BigDecimal.ZERO);
+    BigDecimal overExpended = expended.add(awaitingPayment).subtract(credited)
```

Comment:
For clarity I would prefer to see `expended.subtract(credited).add(awaitingPayment)` instead (keeping expended and credited together).

Reply:
I agree, thanks. Fixed